### DISCUSSION TITLE
getting @prisma/client not initiate error on deployment over vercel

### DISCUSSION THREAD

**Discussion by divyanshdixit** [prisma/prisma#24212] [Q&A]
### Question

I was trying to deploy next js app on vercel, but after deploying successfully completed when i try to access my api , it shows below error and I've done mostly everything to resolve it. 
I've tried every solution available on internet. but don't know why i'm getting same error again and again.
Error: @prisma/client did not initialize yet.

### How to reproduce (optional)

<img width="1098" alt="image" src="https://github.com/prisma/prisma/assets/39085144/9335911b-a18e-4bf0-bd74-8075d12be0fc">

<img width="927" alt="image" src="https://github.com/prisma/prisma/assets/39085144/db82f6b1-3b06-4144-9d50-dd201dcc6551">

<img width="661" alt="image" src="https://github.com/prisma/prisma/assets/39085144/cc2ef8c1-4c14-43d6-b929-ab9a2bebef38">

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @divyanshdixit 👋 

Thank you for raising this question.

This error typically occurs when Prisma Client is not properly generated before your application starts. Here are a few things you can try to resolve this issue:
Run Prisma Generate: Make sure to run prisma generate before starting your application. This command generates the Prisma Client code. You can add it as a `postinstall` script in your package.json file to ensure it runs every time dependencies are installed. For example:
```
"scripts": {
  "postinstall": "prisma generate"
}
```
Do take a look at this [section](https://www.prisma.io/docs/orm/prisma-client/deployment/serverless/deploy-to-vercel) on deploying to vercel.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
